pmm/mysql_queries.go

```diff
@@ -22,91 +22,104 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"strconv"
+	"os"
 	"strings"
 	"time"
 
 	consul "github.com/hashicorp/consul/api"
 	"github.com/percona/kardianos-service"
 	"github.com/percona/pmm/proto"
-	protocfg "github.com/percona/pmm/proto/config"
+	pc "github.com/percona/pmm/proto/config"
 )
 
+// MySQLQueriesFlags MySQL Queries specific flags.
+type MySQLQueriesFlags struct {
+	QuerySource string
+	// slowlog specific options.
+	RetainSlowLogs          int
+	DisableSlowLogsRotation bool
+}
+
+// MySQLQueriesAddResult is result returned by AddMySQLQueries.
```

Comment:
comment on exported type MySQLQueriesResult should be of the form "MySQLQueriesResult ..." (with optional leading article)